packages/bpk-component-text/src/BpkText.module.scss

```diff
@@ -144,4 +159,10 @@
   &--editorial-3 {
     @include typography.bpk-editorial-3;
   }
+
+  @each $color-name, $color-value in $text-colors {
+    &--#{$color-name} {
+      color: $color-value;
+    }
```

Comment:
Use Sass loops - [Sass: @each](https://sass-lang.com/documentation/at-rules/control/each/#with-maps) to generate the classnames